packages/python-pulp-rpm/python-pulp-rpm.spec

```diff
@@ -35,7 +35,7 @@ Requires:       python%{python3_pkgversion}-libcomps >= 0.1.15
 Conflicts:      python%{python3_pkgversion}-libcomps >= 0.2
 Requires:       python%{python3_pkgversion}-productmd >= 1.33
 Requires:       python%{python3_pkgversion}-pulpcore < 3.15
-Requires:       python%{python3_pkgversion}-pulpcore >= 3.14.5
+Requires:       python%{python3_pkgversion}-pulpcore >= 3.14.11
```

Comment:
can you also update the `aiohttp-xmlrpc` Requirement?

Reply:
Done, added on conflict as well